### ISSUE TITLE
HDR Support

### ISSUE THREAD

**Issue by rickomax** [StbSharp/StbImageLib#1]
Hi!

HDR support is missing on this safe version?

**Comment by Cyphall** [comment]
I'm also interested in this.

**Comment by rds1983** [comment]
Yeah, I was too lazy to properly convert this method to safe code: https://github.com/StbSharp/SafeStbImageSharp/blob/5d44c727c2e92b0992f970a053d75904e385f1e4/src/Utility/Conversion.cs#L19
